apps/ftu/build/build.js

```diff
@@ -16,11 +16,17 @@ FtuAppBuilder.prototype.setOptions = function(options) {
   this.gaia = utils.gaia.getInstance(options);
   this.gaia.stageDir = this.stageDir;
   this.gaia.gaiaDir = options.GAIA_DIR;
+  this.raptorMode = options.RAPTOR === '1';
 };
 
 FtuAppBuilder.prototype.generateManifest = function() {
   var manifestObject = importBuild.generateManifest(this.webapp, this.gaia);
   var file = utils.getFile(this.stageDir.path, 'manifest.webapp');
+
+  if (this.raptorMode) {
```

Comment:
Add short comment like 'remove role:system so FTU gets a homescreen icon to allow raptor/marionette to launch it' as its a bit indirect and non-obvious